src/pipecat/services/hailuo.py

```diff
@@ -0,0 +1,195 @@
+from typing import AsyncGenerator, Optional
+import aiohttp
+import json
+from loguru import logger
+from pydantic import BaseModel
+import time
+
+from pipecat.frames.frames import (
+    ErrorFrame,
+    Frame,
+    TTSAudioRawFrame,
+    TTSStartedFrame,
+    TTSStoppedFrame,
+)
+from pipecat.services.ai_services import TTSService
+from pipecat.transcriptions.language import Language
+
+class HailuoTTSService(TTSService):
```

Comment:
Rename to `HailuoHttpTTSService` in case they add a more efficient one (e.g. websockets) in the future. This is the convention we currently use.